src/protos/io/defang/v1/fabric.proto

```diff
@@ -364,7 +364,10 @@ message Event {
 
 message PublishRequest { Event event = 1; }
 
-message SubscribeRequest { string service = 1; }
+message SubscribeRequest { 
+  google.protobuf.Timestamp since = 1;
```

Comment:
I'm surprised to see "since" here. Seems odd to me to subscribe from an old point in time?

Reply:
yes, I am considering taking it out. it's an artifact of having to make two loki queries. One for logs between the "since" and the current time and the current time+ log stream.
I think I'll remove it and just set a current - 10sec for any possible missed deployment logs.

Reply:
After discussion, backend will make a call to out k/v store to get the initial state values and bypass the need for two loki calls.